fil_fungible_token/src/receiver/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod types;
```

Comment:
TODO: library level code for the receiving actor (don't think there'll be _much_ boilerplate tho)

maybe receiver module should be it's own package?